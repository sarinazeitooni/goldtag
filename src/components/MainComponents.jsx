import {Fragment} from "react";
import MainSwipper from './swiper/mainswipper';
import Features from './features/features';
import Goldtags from './goldtags/goldtags';
import Category from './category/category';
import Social from './social/social';
function MainComponents() {
    return (
        <Fragment>
            <MainSwipper/>
            <Features/>
            <Goldtags/>
            <Category/>
            <Social/>
        </Fragment>
    );
}
export default MainComponents;