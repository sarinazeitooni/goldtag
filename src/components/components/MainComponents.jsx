import MainSwipper from './swiper/mainswipper';
import Features from './features/features';
import Goldtags from './goldtags/goldtags';
import Category from './category/category';
import Social from './social/social';
function MainComponents() {
    return (
        <div>
            <MainSwipper/>
            <Features/>
            <Goldtags/>
            <Category/>
            <Social/>
        </div>
    );
}
export default MainComponents;