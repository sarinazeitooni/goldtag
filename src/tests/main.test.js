import React from 'react';
import { shallow } from 'enzyme';
import MainComponents from "../components/MainComponents";
import MainSwipper from "../components/swiper/mainswipper";
import Features from "../components/features/features";
import Goldtags from "../components/goldtags/goldtags";
import Category from "../components/category/category";
import Social from "../components/social/social";
describe("main component" , ()=>{
    it("should match snap shot" , ()=>{
        const wrapper = shallow(<MainComponents/>);
        expect(wrapper).toMatchSnapshot();
    });
   it("should render swiper" , ()=>{
       const wrapper = shallow(<MainComponents/>);
       const swiper = wrapper.find(MainSwipper);
       expect(swiper.exists()).toBe(true);
   });
    it("should render features" , ()=>{
        const wrapper = shallow(<MainComponents/>);
        const feature = wrapper.find(Features);
        expect(feature.exists()).toBe(true);
    });
    it("should render Gold tags" , ()=>{
        const wrapper = shallow(<MainComponents/>);
        const GoldTag = wrapper.find(Goldtags);
        expect(GoldTag.exists()).toBe(true);
    });
    it("should render Category" , ()=>{
        const wrapper = shallow(<MainComponents/>);
        const Categories = wrapper.find(Category);
        expect(Categories.exists()).toBe(true);
    });
    it("should render Social" , ()=>{
        const wrapper = shallow(<MainComponents/>);
        const Socials = wrapper.find(Social);
        expect(Socials.exists()).toBe(true);
    });

});