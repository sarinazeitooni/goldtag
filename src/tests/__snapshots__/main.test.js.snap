// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main component should match snap shot 1`] = `ShallowWrapper {}`;
